skiko/src/jvmMain/kotlin/org/jetbrains/skiko/redrawer/LinuxOpenGLRedrawer.kt

```diff
@@ -68,23 +83,38 @@ internal class LinuxOpenGLRedrawer(
         frameDispatcher.scheduleFrame()
     }
 
-    override fun redrawImmediately() = layer.backedLayer.lockLinuxDrawingSurface {
+    override fun redrawImmediately() {
         check(!isDisposed) { "LinuxOpenGLRedrawer is disposed" }
         update(System.nanoTime())
-        it.makeCurrent(context)
-        draw()
-        it.setSwapInterval(0)
-        it.swapBuffers()
-        OpenGLApi.instance.glFinish()
-        it.setSwapInterval(swapInterval)
+        runBlocking {
```

Comment:
So we'll block AWT thread here? Shall we?

Reply:
Yes, the semantic of this method is to draw content in a blocking way, but without vsync synchronization